backend/aci/server/acl.py

```
@@ -19,15 +19,9 @@ def get_propelauth() -> FastAPIAuth:
 
 
-def validate_user_access_to_org(user: User, org_id: UUID, org_role: OrganizationRole) -> None:
-    # TODO: refactor to use PropelAuth built-in methods
-    org = user.get_org(str(org_id))
-    # TODO: may need to check user_inherited_roles_plus_current_role for project level ACLs
-    if (org is None) or (org.user_is_role(org_role) is False):
-        raise OrgAccessDenied(
-            f"user={user.user_id} does not have access to org={org_id} or "
-            f"does not have the required role={org_role} in the org"
-        )
+def validate_user_access_to_org(user: User, org_id: UUID) -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)